device_operation/device_gemm_xdl_instance_f32_f32_f32_mk_nk_mn.cpp

```diff
@@ -18,6 +19,29 @@ template <ck::index_t... Is>
 using S = ck::Sequence<Is...>;
 
 // Compilation parameters for a[m, k] * b[n, k] = c[m, n]
+#if CK_USE_SPLITK_XDLOPS
```

Comment:
same here, should add split-K GEMM instance in separate *.cpp file

Reply:
done